subtests/docker_cli/run/run_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+from run import run_base
+
+
+class run_tmpfs(run_base):
+    # Verify behavior when multiple --name options passed
+
+    def init_subargs(self):
+        cont = self.sub_stuff["cont"]
+        name_base = cont.get_unique_name()
+        names = []
+        for number in xrange(self.config['names_count']):
```

Comment:
this loop is unrelated to testing --tmpfs option, and related config option as well...